pages/membership.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: page
+title: Membership
+permalink: /membership/
+---
+
+{% if site.accepting_applications == null %}
+<div class='alert' markdown='1'>
+#### We aren't currently accepting applications.
+  
+Join our [general interest mailing list]({{ site.urls.mailing_list }}) to get a notification when we open up membership applications!
+
+(Note: You will be redirected to our Google Groups page. Click "Join group" to get emails.)
+</div>
+{% endif %}
+
+New members join through an application and voting process. If you're interested in being part of Double Union but aren't sure whether you're “good enough,” please apply - we don't evaluate applicants on whether they're “cool” or accomplished. Read on to hear about what we're looking for, and don't let [impostor syndrome](http://geekfeminism.wikia.com/wiki/Impostor_syndrome) stop you.
+
+Along with submitting an application, a qualification for membership is that you've met at least one member who is willing to say you seem like a respectful potential member (such as that you'd respect the [base assumptions]({{ 'base_assumptions' | absolute_url }}), [anti-harassment policy({{ 'policies' | absolute_url }}), and responsibilities of being a member). Please come to events to meet some of us! Or if you'd prefer a quieter opportunity to meet just a couple members over a cup of tea, email [{{ site.emails.join }}](mailto:{{ site.emails.join }}) and we'll set this up.
```

Comment:
`[anti-harassment policy` is missing an end square bracket (`[anti-harassment policy]`)